x-pack/plugins/apm/server/routes/services/get_service_infrastructure.ts

```diff
@@ -57,7 +57,7 @@ export const getServiceInfrastructure = async ({
         },
         hostNames: {
           terms: {
-            field: HOSTNAME,
+            field: HOST_NAME,
```

Comment:
here too

Reply:
I don't know, probably yes. Maybe in a different issue?